src/options/append_to_stream.rs

```diff
@@ -0,0 +1,79 @@
+use crate::event_store::client::shared::Empty;
+use crate::event_store::client::streams::append_req::options::ExpectedStreamRevision;
+use crate::{Credentials, EventData, ExpectedVersion};
+use futures::future::Ready;
+use futures::stream::{Iter, Once};
+use futures::Stream;
+
+#[derive(Clone)]
```

Comment:
**CONCERN:** It think that this might be missing feature, but I don't see the usage of settings like - `requires leader` or `timeouts` (they're available in other clients) - the same suggestion for alternatives.

Reply:
all can be specified through the connection string or connection setting builder.